src/Purebred.hs

```diff
@@ -234,7 +247,18 @@ launch cfg = do
   --
   bchan <- newBChan 32
 
-  cfg' <- processConfig (bchan, b) (pre cfg)
+  -- Create log sink.
+  logSink <- case debugFile opts of
+    Nothing -> pure $ \_ -> pure ()
+    Just fp -> do
+      h <- openFile fp AppendMode
+      hSetBuffering h LineBuffering
+      T.hPutStrLn h $ T.pack "Opened log file"
```

Comment:
Could we put this into a general logging function? The reason why I'm asking is that it would be really good to also have timestamps next to the entries. I would love to add that to our tests and have it shown at the end of the test session. Without timestamps it'll be really hard to figure out when what happened.

Reply:
save for another PR.  feel free to tackle it!

Reply:
Alright